blocks/newslist/newslist.css

```diff
@@ -491,6 +503,14 @@
   .newslist .newslist-pagination-container a.current-page {
     color: var(--link-color);
   }
+
+  .newslist .newslist-container .newslist-item h4 a {
+    font-size: 20px;
+  }
+
+  .newslist .newslist-container .newslist-item .newslist-item-description {
```

Comment:
@veejayricana these selectors are already there in line 28 and 39, please move these styles under the same selector, rather than duplicating the selector to the end of the file.

Reply:
@ravkiran, I think this change is for min-width:1000px not for small view port.